app/controllers/admin/access_controls_controller.rb

```diff
@@ -77,4 +97,29 @@ def assign
   private def set_access_control
     @access_control = access_control_scope.find(params[:id].to_i)
   end
+
+  private def access_controls
+    @access_controls ||= AccessControl.all
```

Comment:
I wonder if we should take this opportunity to add a `visible_to(user)` scope to `AccessControl` even if it just returns `all` for now.